modules/admin-ui/src/main/webapp/scripts/shared/partials/modals/event-details.html

```diff
@@ -416,55 +416,58 @@ <h2 translate="EVENTS.EVENTS.DETAILS.HEADER" translate-values="{{titleParams}}">
                 <table class="main-tbl">
                   <thead>
                     <tr>
-                      <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.ID"><!-- ID --></th>
-                      <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.TYPE"><!-- Type --></th>
-                      <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.STATE"><!-- State --></th>
-                      <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.SUBMITTED">
-                        <!-- Submitted -->
+                      <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.TITLE"><!-- Title --></th>
+                      <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.SUBMITTED_BY">
+                        <!-- Submitted By -->
                       </th>
+                      <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.SUBMITTED_AT">
+                        <!-- Submitted At -->
+                      </th>
+                      <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.STATUS"><!-- Status --></th>
                       <th translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.ACTIONS" ng-if="$root.userIs('ROLE_UI_EVENTS_DETAILS_WORKFLOWS_EDIT')" class="fit"><!-- Actions --></th>
-                      <th class="medium"></th>
                     </tr>
                   </thead>
                   <tbody>
                     <tr ng-repeat="item in workflows.entries|orderBy:'submitted':true track by $index">
-                      <td>{{ item.id }}</td>
-                      <td>{{ item.workflow }}</td>
+                      <td>{{ item.title }}</td>
+                      <td>{{ item.submittedBy }}</td>
+                      <td>{{ item.submittedAt | localizeDate : 'dateTime' : 'medium' }}</td>
                       <td>{{ item.status | translate }}</td>
-                      <td>{{ item.submitted | localizeDate : 'dateTime' : 'medium' }}</td>
-                      <td ng-if="$root.userIs('ROLE_UI_EVENTS_DETAILS_WORKFLOWS_EDIT')" >
+                      <td>
+                        <a class="more" title="{{ 'EVENTS.EVENTS.DETAILS.WORKFLOWS.TOOLTIP.DETAILS' | translate }}"
```

Comment:
This is now quite inconsistent with all other tabs since it's the only one using an icon in an action column to get to a detailed view. Everywhere else a `Details` link is used.

Reply:
Agree, but in this table we have an "Actions" column with other actions (which we don't find in the other places that use "Details"). This again is consistent with the main tables.

Reply:
This would maybe ba consistent if it would also behave consistently in that it would open a modal dialog like the icon in the main table does. But here, it looks like the icon in the main table and behaves like the “Details” links used consistently in all of the dialogs ;-)